.github/workflows/aws.yml

```diff
@@ -1,50 +1,43 @@
-# Basic workflow with Actions
+# Terraform Action to test Cloud Databases with test-harness
 
 name: AWS Cloud Database Test Execution
 on:
   # Triggers the workflow on push & pull request events for the main branch. Also allows for manual triggers
-  # push:
-  #   branches: [ main ]
-  # pull_request:
-  #   branches: [ main ]
-  # schedule:
-  #   - cron: '0 12 * * *' # Execute every day at noon
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+  schedule:
+    - cron: '0 12 * * *' # Execute every day at noon
```

Comment:
Github actions runners are UTC. This is 0700 CST.

Reply:
> Github actions runners are UTC. This is 0700 CST.

This was copied directly from the existing main.yml.

Reply:
The comment mentions noon and this is not noon. Pointing it out.